include/WCSimPrimaryGeneratorAction.hh

```diff
@@ -67,12 +84,13 @@ private:
 
   // These go with jhfNtuple
   G4int mode;
-  G4int vtxvol;
-  G4ThreeVector vtx;
+  G4int nvtxs;
+  G4int vtxsvol[900];
```

Comment:
Can the 900 value be taken from a new constant? This applies to a few files (e.g. jhfNtuple.h). It'll make it easier to update if/when overflows happen. Of course, vectors are an alternative way to do things